packages/web-components/@groupby/elements-sayt/test/unit/sayt.test.ts

```diff
@@ -1085,4 +1087,41 @@ describe('Sayt Component', () => {
       expect(result).to.be.an.instanceof(TemplateResult);
     });
   });
+
+  describe('updateSearchboxInputTerm()', () => {
+    let updateSearchTerm;
+    let group;
+
+    beforeEach(() => {
+      updateSearchTerm = spy();
+      group = sayt.group = 'group-1';
+      stub(sayt, 'querySelector')
+        .withArgs('[data-gbe-ref="autocomplete"]')
+        .returns({ updateSearchTerm, group });
```

Comment:
`group` being on autocomplete isn't relevant to this function and can be removed from the stub.